client/src/pages/Results/ResultPage.tsx

```diff
@@ -250,9 +251,9 @@ const ResultPage = () => {
   const hasFilteredResults = filteredResults.length > 0
 
   return (
-    <>
+    <Box sx={{ minHeight: '100vh', display: 'flex', flexDirection: 'column' }}>
       <Header />
```

Comment:
I'm torn about the fact that we specify this on every page and will need to do this for future pages. I opted to do it this way because there was an issue I was working on one time for a different project where I didn't want the same "100vh container, header, footer" pattern. But should I make it so we have to repeat this everywhere on every new page instead of a possible one-off for a page where we may not want it?
🤔 
@jsstevenson do you have any thoughts?